express/properties/structural/symmetry_symbol_point_group.py

```diff
@@ -0,0 +1,18 @@
+from express.properties.non_scalar import NonScalarProperty
+
+
+class SymmetrySymbolPointGroup(NonScalarProperty):
```

Comment:
Both of these are pretty close a ScalarProperty too, maybe we should use that one instead?